analyzers/tests/SonarAnalyzer.Test/Rules/RouteTemplateShouldNotStartWithSlashTest.cs

```diff
@@ -45,8 +45,9 @@ public class RouteTemplateShouldNotStartWithSlashTest
 
     [TestMethod]
     public void RouteTemplateShouldNotStartWithSlash_CS() =>
-        builderCS
-            .AddPaths("RouteTemplateShouldNotStartWithSlash.AspNetCore.cs")
+    builderCS
+            .AddPaths("RouteTemplateShouldNotStartWithSlash.AspNetCore.cs", "RouteTemplateShouldNotStartWithSlash.AspNetCore.PartialAutogenerated.cs")
+            .WithConcurrentAnalysis(false)
```

Comment:
This is needed otherwise when creating the copy concurrent file it does not seem to transfer the `autogenerated` comment.